srcpkgs/triangle/files/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright 1993, 1995, 1997, 1998, 2002, 2005
+Jonathan Richard Shewchuk
+2360 Woolsey #H
+Berkeley, California  94705-1927
+jrs@cs.berkeley.edu
+
+This program may be freely redistributed under the condition that the
+copyright notices (including this entire header and the copyright    
+notice printed when the `-h' switch is selected) are not removed, and 
+no compensation is received.  Private, research, and institutional   
+use is free.  You may distribute modified versions of this code UNDER
+THE CONDITION THAT THIS CODE AND ANY MODIFICATIONS MADE TO IT IN THE
+SAME FILE REMAIN UNDER COPYRIGHT OF THE ORIGINAL AUTHOR, BOTH SOURCE  
```

Comment:
> MODIFICATIONS ... REMAIN UNDER COPYRIGHT OF THE ORIGINAL AUTHOR

If I understand this correctly, this is try to take over rights to modifications from author of modification. Unacceptable even as restricted.

Reply:
So it should go into nonfree?

P.S.: Sorry for all the noise, still learning.